bodhi/server/templates/update.html

```diff
@@ -341,6 +344,8 @@
     $("#tab-automatedtests").append(" <span class='fa fa-spinner fa-spin'></span>");
 
     for (index = 0, len = builds.length; index < len; ++index) {
+      request_results(base_url+"results/latest?original_spec_nvr="
+        +builds[index]+"&testcases=org.centos.prod.ci.pipeline.complete");
       request_results(base_url+"results/latest?item="+builds[index]+"&type=koji_build&testcases:like=dist.*");
```

Comment:
Just a small thing - please add `since=` parameter in ISO8601 format (UTC without tz) to limit the search for the `request_results`. Bodhi IIRC knows when the update was created (or something like that), so set it to that (or any other reasonable) value, so it does not need to crawl the whole db in order to find that "for some testcases, there is no result". Thanks!

Reply:
The builds were tested in centosci *before* the update was created, so I'm afraid there's no easy value to pass to `since=`. We could limit showing results to builds tested in the past 2 weeks, or a month, but it might be... limiting. Probably best to introduce that if we have performance issues.

We use `since=` when requesting results for `type=bodhi_update`, but we don't use it when requesting results for `type=koji_build` (see, another nice example of why conventions are good). And this is the same case as koji builds, I believe.

Reply:
Sure, that's where "something reasonable" goes. We already do have perf issues, and I'll be adding a "data since last month" defualt constraint to resultsdb anyway real-time-soon-now.

Reply:
Doesn't using the /latest endpoint limit the results by default? (which should mean it's not scrolling the entire DB)

Reply:
@pypingou I understand the confusion, but no. `/latest` limits the data _output_, but say you query for `?item=foo&testcases=moo,bar`. It does something like `select * from results where item=foo and testcase=moo` + `select * from results where item=foo and testcase=bar` (disclaimer: this is far more complicated in fact, but on a high level is what is happening).
In this example, if the `foo/bar` pair is not present, the only way (for any db) to say "there is none" is to go through all the data (or a subset of data limited by index).
So the queries are limited in some way, but between the amount of data in resultsdb, the insufficient hardware the db has, and the postgres' query planner, looking for sparse data is quite slow. Thus narrowing the interval can help.
This is not needed, nor required, but take it as a tip from an insider :)

Reply:
@rajcze So what do you think it's the reasonable value here? Is it better to fall back to the default value you'll be soon adding (so e.g. a month old bodhi updates will stop displaying results), or is it better to use a custom `since=` with a timestamp e.g. 2-4 weeks prior to the update creation? (The builds might be available and tested easily 2 weeks before the maintainer decides to submit them as an update).

Btw, we should use the same approach for `type=koji_build` query, not just this centosci one.

Reply:
While I think this is a good question and a chance we should look into (indeed not just for the Atomic CI but likely for other), do we want to bring this in another PR?

Also, the since = 2 weeks before the update means that in a year, resultsdb may still have to go through 1 year + 2 weeks of data. Would an upper bound help?
Something like since <update - 2 weeks> until <update + 1 week> ?